lua/lazyvim/plugins/extras/coding/mini-snippets.lua

```diff
@@ -0,0 +1,159 @@
+--[[
+
+This extra for mini.snippets activates the following:
+1. During completion with either cmp or blink, expand lsp snippets
+2. To use other snippets: Press <c-j> in insert mode
+
+Note that there is no completion source. 
+Other snippets(custom or from friendly-snippets) are handled by pressing <c-j> in insert mode.
+
+In default LazyVim, neither cmp nor blink define the default mappings used by mini.snippets:
+- "<c-j>" to expand
+- "<c-l>" to jump next (dynamically created when in snippet context)
+- "<c-h>" to jump previous (dynamically created when in snippet context)
+
+It's difficult to have jump_next or jump_previous working in all cases when mapped to "<tab>"/"<s-tab>".
```

Comment:
The best first approximation is [present in 'mini.snippets' docs](https://github.com/echasnovski/mini.nvim/blob/733afb94ed88d32b839a644dc137604691e46032/doc/mini-snippets.txt#L407). But yes, it is indeed somewhat difficult to make a flexible mapping for a distro.

Reply:
I tried the extra in its current state with the following:

```lua
mappings = {
  -- keep the default expand, a global mapping
  jump_next = "<tab>",
  jump_prev = "<s-tab>"
```

No luck with `blink.cmp` however. As there are no sources yet I did not investigate any further.